docker/docker-compose.yml

```diff
@@ -19,6 +19,9 @@ services:
       - NODE_ENV
       - API_HOST
       - STATUS_PORT
+      - STATUS_URL
+      - POSTS_URL=http://posts
```

Comment:
This shouldn't need to be defined, just forward what is already in the environment.

Reply:
But in the environment `POSTS_URL` is `http://localhost/v1/posts` but I need to call `http://posts` to be able to talk with post microservices.

Reply:
Understood.  See my comment above.  Let's do what planet does.